force-app/main/triggers/classes/HOT_UserHandlerTest.cls

```diff
@@ -31,12 +36,61 @@ private class HOT_UserHandlerTest {
             System.assertEquals(1, serviceResources.size());
         }
     }
+
+    //Tests if a user that was changed to HOT Ansatt Tolk has a Service Resource created.
+    @isTest
+    static void changedToAnsattTolk() {
```

Comment:
Mye av denne koden er for å opprette en community.bruker. Logikken vi har for ansatt-tolk er for interne brukere og du trenger da ikke noe annet en en vanlig bruker som bytter profil til HOT Tolk Ansatt.